### PR TITLE
Add tenant name to page name

### PR DESCRIPTION
**Pull request by djnunez-aot**
Issue #2160 

-add tenant name to page name on loading tenant

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

**Review by saravanpa-aot — DISMISSED** [review]
There is a localisation file which should be used for this..

File: met-web/src/App.tsx [hunk 1]
```diff
@@ -94,10 +94,12 @@ const App = () => {
 
     const getTranslationFile = async () => {
         try {
-            const translationFile = await import(`./locales/${language}/${tenant.id}.json`);
+            const translationFile = await import(`./locales/${language}/${basename}.json`);
+            document.title = translationFile?.header?.title || document.title;
```
Comment (jadmsaadaot) [inline met-web/src/App.tsx:98]: ok so maybe we need to leverage React Helmet to do something like this, it's probably cleaner. Can you check the library

File: met-web/src/App.tsx [hunk 2]
```diff
@@ -94,10 +94,12 @@ const App = () => {
 
     const getTranslationFile = async () => {
         try {
-            const translationFile = await import(`./locales/${language}/${tenant.id}.json`);
```
Comment (jadmsaadaot) [inline met-web/src/App.tsx:97]: also why the change from tenant.id to basename, this overrides my changes in another PR for the single tenancy/ multi tenancy task

File: met-web/src/App.tsx [hunk 3]
```diff
@@ -95,9 +97,13 @@ const App = () => {
     const getTranslationFile = async () => {
         try {
             const translationFile = await import(`./locales/${language}/${tenant.id}.json`);
+            const title = translationFile?.header?.title || document.title;
```
Comment (jadmsaadaot) [inline met-web/src/App.tsx:100]: that's not how we get values from the translationFile. Check     const { t: translate } = useAppTranslation(); in the landingComponent for example

**Review by jadmsaadaot — CHANGES_REQUESTED** [review]
use const { t: translate } = useAppTranslation();

**Review by jadmsaadaot — APPROVED** [review]

**Pull request merged**